examples/jsm/objects/BatchedMesh.js

```diff
@@ -10,16 +10,16 @@ import {
 } from 'three';
 
 const ID_ATTR_NAME = '_batch_id_';
-const _identityMatrix = new Matrix4();
-const _zeroScaleMatrix = new Matrix4().set(
+const _identityMatrix = /* @__PURE__ */ new Matrix4();
+const _zeroScaleMatrix = /* @__PURE__ */ new Matrix4().set(
```

Comment:
Nit: I don't think we need these annotations in the addons. three-shaking also works without them.

Reply:
Okay! Removed